pkg/kudoctl/util/repo/repo.go

```diff
@@ -4,13 +4,19 @@ import (
 	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
 )
 
-// Entry represents a collection of parameters for framework repository
-type Entry struct {
-	Name      string `json:"name"`
+// RepositoryConfiguration represents a collection of parameters for framework repository
+type RepositoryConfiguration struct {
 	LocalPath string `json:"localPath"`
 	URL       string `json:"url"`
 }
 
+func NewRepositoryConfiguration() *RepositoryConfiguration {
+	return &RepositoryConfiguration{
+		LocalPath: "$HOME/.kudo/repository", // this won't work on windows
+		URL: "https://kudo-test-repo.storage.googleapis.com",
```

Comment:
we should make this `kudo-repo` as the official one or so.

I have:

- kudo-frameworks
- kudo-registry
- kudo-repo
- kudo-repository
- kudo-test-repo
- kudobuilder

as options already reserved for us.